.github/workflows/benchmarks.yml

```diff
@@ -39,7 +39,9 @@ jobs:
           comment_tag: check-size # ensures we create or update one size comment
 
       - name: Build
-        run: npm run build
+        run: |
+          cd packages/benchmarks
+          npm run build
```

Comment:
Is this change expected when updating chromedriver?

Reply:
No, I added this in because I was already touching the benchmark code. Currently the build step takes a couple minutes, and this should drop that down a good bit